### PR TITLE
Fix bash if space syntax

### PR DESCRIPTION
**Pull request by jennydaman**
Currently this error is produced

```
/cromwell_root/script: line 31: [-z: command not found
```

### CODE REVIEW

File: tasks/broad/Alignment.wdl [hunk 1]
```diff
@@ -54,7 +54,7 @@ task SamToFastqAndBwaMemAndMba {
     set -o pipefail
     set -e
 
-    if [-z ${BWA_VERSION}]; then
+    if [ -z ${BWA_VERSION}]; then
```
Comment (gbggrant) [inline tasks/broad/Alignment.wdl:57]: Might as well put a space on both sides 
```suggestion
    if [ -z ${BWA_VERSION} ]; then
```
I have also read that the double bracket syntax is supposed to be better. '[[' vs '['

**Review by gbggrant — APPROVED** [review]
Thank you!

**Review by jessicaway — APPROVED** [review]

File: tasks/broad/Alignment.wdl (hunk 1 above)
Comment (jennydaman) [reply tasks/broad/Alignment.wdl]: Thanks for catching that! `}]` is still invalid syntax, I made this PR in a hurry.

`[[` is a superset of `[`. I don't think it's necessary.

**Comment by jessicaway** [comment]
retest this please

**Comment by jessicaway** [comment]
ok to test

**Comment by jessicaway** [comment]
@jennydaman, thank you for contributing this PR. Unfortunately we do not yet have the ability to safely test PRs from users outside our team. This is an issue we plan to address as soon as possible, but in this case I have created [a separate PR](https://github.com/broadinstitute/warp/pull/178) with your changes to allow for testing

**Comment by jennydaman** [comment]
I've noticed, though the routine seems excessive for insertion of two whitespace characters.

**Comment by jessicaway** [comment]
Yes, we are still working on balancing thorough testing with faster iteration time. It can be especially painful for small changes like this. Hopefully we will be able to come up with a simpler workflow for these small changes soon

**Comment by jessicaway** [comment]
Replaced by [this PR ](https://github.com/broadinstitute/warp/pull/178)

**Pull request closed without merging**